src/app/pages/profile/transaction-information/transaction-information.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { TransactionInformationService } from '../../../services/profile/transaction-information/transaction-information.service';
+import { APPCONFIG } from 'src/app/config';
+
+@Component({
+  selector: 'app-transaction-information',
+  templateUrl: './transaction-information.component.html',
+  styleUrls: ['./sass/transaction-information.component.scss']
+})
+export class TransactionInformationComponent implements OnInit {
+  base_url = APPCONFIG.base_url;
+  transacationUrl = `${this.base_url}/transactions/`;
+  sucessTransactions: any[] = [];
+  inProgressTransactions: any[] = [];
+  results: any[] = [];
+  resultSucces: any[] = [];
```

Comment:
`resultSuccess` & `resultProgress` are unused here

Reply:
I worked on that, thanks